docs/pages/management/guides/docker.mdx

```diff
@@ -109,9 +114,8 @@ Move your `license.pem` file to `~/teleport/data`.
 # Generate a sample Teleport config and write it to the local config directory.
 # This container will write the config and immediately exit--this is expected.
 $ docker run --hostname localhost --rm \
-  --entrypoint=/bin/sh \
-  -v ~/teleport/config:/etc/teleport \
-  (=teleport.latest_ent_docker_image=) -c "teleport configure > /etc/teleport/teleport.yaml"
+  --entrypoint=/usr/local/bin/teleport \
+  ${TELEPORT_DOCKER_IMAGE} configure > ~/teleport/config/teleport.yaml
```

Comment:
Should we change line 124 as well?
It still shows the markdown placeholder and not the env var defined above
```
 (=teleport.latest_ent_docker_image=)
```

Reply:
Done